libs/model/package.json

```diff
@@ -20,26 +20,18 @@
     "test": "yarn build && NODE_OPTIONS='--import tsx/esm' NODE_ENV=test mocha 'test/**/*.spec.ts'"
   },
   "dependencies": {
-    "@hicommonwealth/core": "*",
-    "@hicommonwealth/chains": "*",
-    "@hicommonwealth/schemas": "*",
-    "@hicommonwealth/shared": "*",
-    "@hicommonwealth/logging": "*",
+    "@anatine/zod-mock": "^3.13.3",
     "@cosmjs/stargate": "^0.31.3",
     "@cosmjs/tendermint-rpc": "^0.31.3",
+    "@faker-js/faker": "^8.4.1",
+    "mocha-steps": "^1.3.0",
     "node-object-hash": "^3.0.0",
     "pg": "^8.11.3",
     "sequelize": "^6.32.1",
+    "umzug": "^3.7.0",
     "web3": "1.8.2",
     "web3-core": "1.8.2",
     "web3-utils": "1.8.2",
     "zod": "^3.22.4"
-  },
-  "devDependencies": {
-    "@anatine/zod-mock": "^3.13.3",
-    "@faker-js/faker": "^8.4.1",
-    "mocha-steps": "^1.3.0",
-    "tsx": "^4.7.2",
```

Comment:
`tsx` is still a dependency for running tests so it should still be added to dependencies no?

Reply:
We are adding it to the dependencies now. But tests are run before the app is built, so that is why it is fine to be in devDependencies.

But I just moved everything to dependencies to unblock the release. We can fine-tune what we need to move back to devDependencies in the future

Reply:
No but I mean you didn't move `tsx` to dependencies so now it's not install if you `yarn install` only in `libs/model`